daemons/controld/controld_execd.c

```diff
@@ -1750,11 +1774,12 @@ do_lrm_invoke(long long action,
     const char *operation = NULL;
     ha_msg_input_t *input = fsa_typed_data(fsa_dt_ha_msg);
     const char *user_name = NULL;
-    const char *target_node = NULL;
+    const char *target_node = lrm_op_target(input->xml);
+    bool targeting_all_nodes = lrm_op_targets_all_nodes(input->xml);
```

Comment:
I'm thinking we shouldn't functionize this or declare it function-wide, since most ops consider NULL target to mean the local node. I believe only reprobe considers NULL to mean all nodes (and interestingly, the debug message in pcmk_controld_api_reprobe() gets it wrong).

Reply:
Ah, okay.

I had also thought about doing something like `lrm_op_target_raw()`, which in light of your comment might be more broadly useful. But for now it's simple enough not to functionize.